services/ui-src/src/components/fields/ChoiceListField.tsx

```diff
@@ -48,14 +48,22 @@ export const ChoiceListField = ({
   const { full_name, state, userIsAdmin } = useUser().user ?? {};
   // get form context and register field
   const form = useFormContext();
-  form.register(name);
+  const fieldIsRegistered =
+    typeof form.getValues() == "object" ? name in form.getValues() : false;
 
-  const shouldDisableChildFields =
-    userIsAdmin && report?.formTemplate.adminDisabled;
+  const shouldDisableChildFields = userIsAdmin && !!props?.disabled;
 
   // set initial display value to form state field value or hydration value
   const hydrationValue = props?.hydrate;
 
+  useEffect(() => {
+    if (!fieldIsRegistered) {
+      form.register(name);
+    } else {
+      form.trigger(name);
+    }
+  }, []);
+
```

Comment:
You will see in this a lot. This is so when you go forward and backward through a form it'll light up a validation message. Specifically:

First render it wont show anything
If you tab on and tab off it'll throw an error
If you _never touch_ a field and click continue or go to a different page and come back it'll throw an error
It won't wipe an error if that field has been rendered before. Before you could go forward 2 pages and come back and the error would be gone.